packages/bff/src/utilities/errorMappers.ts

```diff
@@ -106,6 +107,28 @@ export const getClientUsersErrorMapper = (
     .with("userNotFound", () => HTTP_STATUS_NOT_FOUND)
     .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
 
+export const getPrivacyNoticeErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("privacyNoticeNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("privacyNoticeNotFoundInConfiguration", () => HTTP_STATUS_NOT_FOUND)
+    .with("dynamoReadingError", () => HTTP_STATUS_INTERNAL_SERVER_ERROR)
+    .otherwise(() => HTTP_STATUS_INTERNAL_SERVER_ERROR);
+
+export const acceptPrivacyNoticeErrorMapper = (
+  error: ApiError<ErrorCodes>
+): number =>
+  match(error.code)
+    .with("privacyNoticeNotFound", () => HTTP_STATUS_NOT_FOUND)
+    .with("privacyNoticeNotFoundInConfiguration", () => HTTP_STATUS_NOT_FOUND)
+    .with("privacyNoticeVersionIsNotTheLatest", () => HTTP_STATUS_CONFLICT)
```

Comment:
I don't see this mapped as a `409` in Scala, maybe we could leave it as a `500`?